openapi-integration-tests/src/test/resources/ballerina_sources/project_openapi_info_with_service_contract_type/service_file.bal

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/openapi;
+
+@openapi:ServiceInfo {
+    version: "1.0.0",
+    title: "Pet store",
+    description: "API system description",
+    email: "sumudu@abc.com",
+    contactName: "sumudu",
+    contactURL: "http://mock-api-contact",
+    termsOfService: "http://mock-api-doc",
+    licenseName: "ABC",
+    licenseURL: "http://abc.com"
+}
+@http:ServiceConfig {basePath: "/v1"}
+type OASServiceType service object {
```

Comment:
Shall we add a test where we have the `@openapi:ServiceInfo` in both service contract type and the implementation? Ideally we should not allow it. I just wanted to check the current functionality

Reply:
When both the service contract type and its implementation are present, the tool generates two OpenAPI specifications (OAS), differing only in the info sections. The ideal behaviour should be to provide an error message, avoiding the addition of annotations for both the contract type and service, as we plan to address this in [this issue](https://github.com/ballerina-platform/ballerina-library/issues/7336). I added a test to verify the existence of two files; this test will fail once the issue is resolved.